Rakefile

```diff
@@ -51,13 +51,29 @@ end
 
 desc 'Generate data for a news digest. Default timeframe is a week since today. For other period, use "since" argument: since="jul 4"'
```

Comment:
```suggestion
desc "Generate data for a news digest. Default timeframe is since last update (determined automatically). For a custom date, use "since" argument: rake whatsnew since='jul 4'"
```